src/WebJobs.Script/DependencyInjection/ScriptStartupTypeLocator.cs

```diff
@@ -81,16 +81,16 @@ public async Task<IEnumerable<Type>> GetExtensionsStartupTypesAsync()
             var bundleConfigured = _extensionBundleManager.IsExtensionBundleConfigured();
             bool isLegacyExtensionBundle = _extensionBundleManager.IsLegacyExtensionBundle();
             bool isPrecompiledFunctionApp = false;
-            bool isDotnetIsolatedApp = false;
 
             // dotnet app precompiled -> Do not use bundles
             var workerConfigs = _languageWorkerOptions.CurrentValue.WorkerConfigs;
+            ImmutableArray<FunctionMetadata> functionMetadataCollection = ImmutableArray.Create<FunctionMetadata>();
```

Comment:
use Enumerable.Empty<FunctionMetadata>() to avoid an unecessary allocation

Reply:
done